pipes/WDL/tasks/tasks_nextstrain.wdl

```diff
@@ -37,21 +39,22 @@ task nextclade_one_sample {
         fi
 
         nextclade run \
-            --input-fasta "~{genome_fasta}" \
-            --input-root-seq ~{default="reference.fasta" root_sequence} \
-            --input-tree ~{default="tree.json" auspice_reference_tree_json} \
-            --input-qc-config ~{default="qc.json" qc_config_json} \
-            --input-gene-map ~{default="genemap.gff" gene_annotations_json} \
-            --input-pcr-primers ~{default="primers.csv" pcr_primers_csv} \
-            --input-virus-properties ~{default="virus_properties.json" virus_properties}  \
+            $DATASET_ARG \
+            ~{"--input-root-seq " + root_sequence} \
+            ~{"--input-tree " + auspice_reference_tree_json} \
+            ~{"--input-qc-config " + qc_config_json} \
+            ~{"--input-gene-map " + gene_annotations_json} \
+            ~{"--input-pcr-primers " + pcr_primers_csv} \
+            ~{"--input-virus-properties " + virus_properties}  \
+            --output-basename "~{basename}" \
```

Comment:
This was also throwing things off for me. Seemed as if it was looking for intermediate files that weren't tagged appropriately. Removing this and explicitly naming the output files below seemed to suffice.

Reply:
Which was throwing things off, `--output-basename`? I think in combination with `--output-all` it works.

I managed to get the `nextclade_one_sample` passing in Github Actions, and I tested the `nextclade_many_samples` task on some recent production data just fine